src/test/java/de/qabel/desktop/hockeyapp/CrashReporterClientTest.java

```diff
@@ -14,7 +14,8 @@
 public class CrashReporterClientTest {
 
     private CloseableHttpClientStub httpClientStub = new CloseableHttpClientStub();
-    private HockeyAppRequestBuilder requestBuilder = new HockeyAppRequestBuilder("1.1", httpClientStub);
+
+    private HockeyAppRequestBuilder requestBuilder = new HockeyAppRequestBuilder("https://rink.hockeyapp.net/api/2/apps/", "1.1", httpClientStub);
```

Comment:
wrap that line or use a shorter url (it's stubbed so feel free to use `"url"` as the hockeyapp base url)